inference_perf/loadgen/load_generator.py

```diff
@@ -177,9 +205,24 @@ def _sigint_handler(self, _signum: int, _frame: Optional[FrameType]) -> None:
         """SIGINT handler that sets interrup_sig flag to True"""
         self.interrupt_sig = True
 
+    def _set_worker_concurrency(self, concurrency_level: int) -> None:
+        # Calculate new concurrency for worker (concurrency_level will always be > 0)
```

Comment:
This logic is fine, alternative proposal if it is more readable.

worker_concurrency = concurrency_level // self.num_workers
remaining = concurrency_level % self.num_workers
for worker in self.workers:
    worker.concurrency = worker_concurrency + 1 if worker.id < remaining else worker_concurrency